packages/nestjs-mongodb/src/mongo.health.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable } from '@nestjs/common';
+import type { HealthIndicatorResult } from '@nestjs/terminus';
+import { HealthCheckError, HealthIndicator } from '@nestjs/terminus';
+import { HealthCheckErrorDto } from '@voiceflow/nestjs-common';
+import { Document } from 'mongodb';
+
+import { MongoService } from './mongo.service';
+
+@Injectable()
+export class MongoHealthIndicator extends HealthIndicator {
+  constructor(private readonly mongoService: MongoService) {
+    super();
+  }
+
+  async isHealthy(key: string): Promise<HealthIndicatorResult> {
+    let isHealthy: boolean;
+    let data: { response: Document } | HealthCheckErrorDto;
+
+    try {
+      const pingResponse = await this.mongoService.db.stats();
```

Comment:
`pingResponse` should also be renamed to `stats` or maybe `statsResponse`

Reply:
makes sense, normalized everything to statsResponse